lib/klaviyo/apis/lists.rb

```diff
@@ -54,18 +54,26 @@ def self.delete_list(list_id)
     # @param :push_tokens [List] push tokens of the profiles to check
     # @return A list of JSON objects of the profiles. Profiles that are
     #   supressed or not found are not included.
-    def self.check_list_subscriptions(list_id, kwargs = {})
+    def self.check_list_subscriptions(list_id, emails = [], phone_numbers = [], push_tokens = [])
       path = "#{LIST}/#{list_id}/#{SUBSCRIBE}"
-      v2_request(HTTP_GET, path, kwargs)
+      params = {
+        "emails": emails,
+        "phone_numbers": phone_numbers,
+        "push_tokens": push_tokens
```

Comment:
Not sure about Ruby but for our python sdk we have these keys as constants at the parent class and then the specific resource classes (lists, profiles, metrics) can all use those constants. Not sure if we're following the same pattern but if so you could move these to that parent class.